serverless.yml

```diff
@@ -19,42 +19,8 @@ provider:
   stage: ${opt:stage, 'prod'}
   region: ${opt:region, 'eu-central-1'}
```

Comment:
do we have cases where we are not in eu-central-1 ?

Reply:
Not that I can see for now. Only times we use something else (`us-east-1`) is for certificates and this kind of stuff but not for what we use serverless with

Reply:
so use 'eu-central-1' directly to remove unnecessary complexity...